engine/src/configuration/applier/timeperiod.cc

```diff
@@ -86,6 +86,9 @@ void applier::timeperiod::add_object(configuration::timeperiod const& obj) {
 
   // Fill time period structure.
   _add_time_ranges(obj.timeranges(), tp.get());
+  if (obj.name() == "Astreinte_Paas_1_avec_feries_2021") {
+    int tutu = 1;
+  }
```

Comment:
I suppose, this is removed in one of the following commits...

Reply:
yes